route/build/src/public_var.f90

```diff
@@ -12,8 +12,6 @@ MODULE public_var
 
   save
 
-  ! ---------- common constants ---------------------------------------------------------------------
```

Comment:
Why is this removed?

Reply:
I removed this because this is duplicated comment with line 20.